Kingfisher/UIImageView+Kingfisher.swift

```diff
@@ -110,6 +111,21 @@ public extension UIImageView {
                          progressBlock: DownloadProgressBlock?,
                      completionHandler: CompletionHandler?) -> RetrieveImageTask
     {
+        var _indicator = UIActivityIndicatorView(activityIndicatorStyle:.Gray)
```

Comment:
Indicator can be a constant. I would also remove the `_` prefix as it doesn't match the current coding style. Any thoughts @onevcat 
```swift
let indicator = UIActivityIndicatorView(activityIndicatorStyle:.Gray)
```